### DISCUSSION TITLE
Changing module name to `github.com/helmfile/helmfile`

### DISCUSSION THREAD

**Discussion by austince** [helmfile/helmfile#98] [Questions] [answered]
Hey there, just wondering if there is a reason the module name is still `github.com/roboll/helmfile` instead of `github.com/helmfile/helmfile`? This makes it impossible to install this version using `go install` if I understand it correctly. Thanks!

**Comment by yxxhero** [comment]
agreed. PR is welcome.

**Comment by yxxhero** [comment, marked as answer]
done.
